README.rst

```diff
@@ -125,16 +78,18 @@ If you want to use the ephys module of BluePyOpt, you first need to install NEUR
 
 And then bluepyopt itself:
 
-```bash
-pip install bluepyopt
-```
+
+.. code-block:: bash
```

Comment:
Why not use the easier ```?

Reply:
That does not work. It is `rst` now, not `md`.

Reply:
Why the change from md to rst?